guide/contribute/illustration-guidelines.md

```diff
@@ -0,0 +1,125 @@
+---
+layout: guide
+title: Illustration guidelines
+description: Tips for creating and preparing visuals for the Bitcoin Design Guide.
+nav_order: 3
+parent: Contribute
+permalink: /guide/contribute/illustration-guidelines/
+image: /assets/images/guide/contribute/illustration-guidelines/illustration-guidelines-preview.jpg
+---
+
+# Illustration guidelines
+
+As this is a design guide, it is only natural that much of our content uses imagery to communicate. Not only is it more appealing, some concepts can also be more effectively communicated visually. Whether it's flow charts, interface mock-ups, interactive prototypes, animations or videos, feel free to explore to find the best way to capture your ideas.
+
+## Creative direction
+
+The basic design of this website was intentionally kept minimal, so authors have more freedom to express their own creative ideas in the content they create. So we encourage experimentation and expression, just keep in mind that the total experience of the guide should still be coherent to readers and allows for easy absorption of the content. Regularly discussing your ideas with others and getting feedback is the best way to ensure we can achieve this. Also make sure to look around and take inspiration from what others have done.
+
+## File folders
+
+All images in the guide are located in the `assets/images/guide` folder. Create a subfolder that matches the URL of your page and place your images there.
+
+For example, the URL for this page is:
+
+`/guide/contribute/illustration-guidelines/`
+
+Images for this page are located in:
+
+`assets/images/guide/contribute/illustration-guidelines/`
+
+## File names
+
+Keep your file names in lowercase and avoid any special characters (including spaces).
+
+Bad:
+
+`My übercool image.PNG`
+
+Good:
+
+`my-ubercool-image.png`
```

Comment:
Let's have standard here. In private key management, Daniel used `Do and Don't` I used the same in the [Content Guideline](https://bitcoin.design/guide/contribute/content-guidelines/#dont). Using heading is good because you can link back to it when we're doing a reviews and show people specific example that's good or bad.

Reply:
Adjusted. Used bold text, as headers created a ton of white space. But since there section is so short, we can use the anchor of the section heading itself.